package.json

```diff
@@ -4,13 +4,15 @@
   "description": "Make CSS easier and more maintainable by using JavaScript",
   "main": "dist/free-style.js",
   "typings": "dist/free-style.d.ts",
+  "module": "dist.es2015/free-style.js",
+  "jsnext:main": "dist.es2015/free-style.js",
   "files": [
     "dist/",
     "LICENSE"
   ],
   "scripts": {
     "lint": "tslint \"src/**/*.ts\"",
-    "build": "rimraf dist/ && tsc",
+    "build": "rimraf dist/ dist.es2015/ && tsc && tsc -P tsconfig-es2015.json",
```

Comment:
A little pedantic, but if we're doing `.` do you mind making it `tsconfig.es2015.json`? 😄